client/src/components/entry.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
```

Comment:
Overall comment for entry.jsx: Great use of createRef and DOM methods to get accurate positioning for your hover.